roles/common/README.md

```diff
@@ -1,41 +1,151 @@
-Role Name
-=========
+# Common Role
 
-The are packages that are generally expected on a server that Ubuntu doesn't
-always have on by default.
+Install all the tools our VMS have in Common
 
-Requirements
-------------
+## Overview
 
-None
+Since we include logrotate in the role, we want other roles to have access to a highly flexible, hierarchical approach to managing log rotation across your infrastructure. It allows you to define logrotate rules at multiple levels of your Ansible inventory, from global defaults to host-specific configurations.
```

Comment:
This file is still the README for the common role, which does a lot more than log rotation. This Overview feels very tightly focused on one topic.